rcl/test/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+if(AMENT_ENABLE_TESTING)
```

Comment:
This check shouldn't be necessary. It is checked before adding the subdirectory.